pydarn/plotting/rtp.py

```diff
@@ -393,14 +408,17 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
         ax.set_xlim([rounded_down_start_time, x[-1]])
         ax.xaxis.set_major_formatter(dates.DateFormatter(date_fmt))
         if ymax is None:
-            ymax = max(y)
+            ymax = np.max(y)
 
         if ymin is None:
-            ymin = min(y)
+            ymin = np.min(y)
 
         ax.set_ylim(ymin, ymax)
 
-        if coord is Coord.SLANT_RANGE:
+        if coord is Coords.SLANT_RANGE:
+            ax.yaxis.set_ticks(np.arange(np.ceil(ymin/100.0)*100,
+                                         ymax+1, yspacing))
+        elif coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
```

Comment:
``` suggestion
        if coord is Coords.SLANT_RANGE or coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
```

they plot the same thing assuming the ticks are good.

Reply:
@carleyjmartin is this the spot?

Reply:
Nah it's later in the minor ticks (L438-444), I'm trying to find the source now, but for reference the minor tick bug from before is now happening in slant range option.

Reply:
Found the issue! Will comment a suggestion!